content/fr/event-streaming.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Diffusion d'événements en continu (Event Streaming)
+status: Completed
+category: concept
+tags: ["méthodologie", "réseau", ""]
+---
+
+La diffusion d'événements en continu (Event Streaming) est une approche où un logiciel envoie des données d'événements d'une application à une autre pour communiquer en continu ce qu'elles font.
+Imaginez un service diffusant tout ce qu'il fait à tous les autres services.
+Chaque activité effectuée par un service est appelée un événement, d'où le terme de diffusion d'événements en continu.
+Par exemple, le NASDAQ reçoit des mises à jour sur les prix des actions et des matières premières chaque seconde.
+Si vous aviez une application surveillant un ensemble spécifique d'actions, vous voudriez recevoir ces informations en quasi temps réel.
```

Comment:
Il faut un trait d'union entre "quasi" et "temps" => "quasi-temps réel" 
Source trouvée chez Larousse
https://www.larousse.fr/dictionnaires/francais/quasi/65549

Reply:
fix in https://github.com/cncf/glossary/pull/2950/commits/c9a79243fb79bd1e0e5744735031a45c3984b7c3